envi/archs/arm/disasm.py

```diff
@@ -218,7 +218,7 @@ def p_dp_imm_shift(opval, va):
 qop_mnem = ('qadd','qsub','qdadd','qdsub')
 smla_mnem = ('smlabb','smlabt','smlatb','smlatt',)
 smlal_mnem = ('smlalbb','smlalbt','smlaltb','smlaltt',)
-smul_mnem = ('smulbb','smulbt','smultb','smultt',)
+smul_mnem = ('smulbb','smultb','smulbt','smultt',)  #bt and tb are backwards. Note xy bits are actually yx
```

Comment:
here is correct.  i'm telling you to swap the letters in the name of the variable in the decoder.